libicu has removed the TRUE definition, switch to using true instead
diff --git a/encoding.c b/encoding.c
index ad4d8a6..68c3840 100644
--- a/encoding.c
+++ b/encoding.c
@@ -1958,7 +1958,7 @@ xmlEncOutputChunk(xmlCharEncodingHandler *handler, unsigned char *out,
 #ifdef LIBXML_ICU_ENABLED
     else if (handler->uconv_out != NULL) {
         ret = xmlUconvWrapper(handler->uconv_out, 0, out, outlen, in, inlen,
-                              TRUE);
+                              true);
     }
 #endif /* LIBXML_ICU_ENABLED */
     else {
